invenio_search/walkers/elasticsearch.py

```diff
@@ -81,6 +81,11 @@ def visit(self, node, left, right):
             keyword = self.map_keyword_to_fields(
                 left, getattr(right, '__search_mode__', 'a')
             )
+            if keyword == ['authors.full_name']:
+                from inspirehep.modules.authors.utils import author_tokenize
+                name_variations = author_tokenize(node.right.value)
+                return {"bool": {"must": [{"terms": {"authors.name_variations": name_variations}}], "should": [
```

Comment:
This then is not accounting for the lower/upper case and accent folding, right? I think it should also be a match.